src/transformation/rgbz.c

```diff
@@ -8,9 +8,12 @@
 #include <limits.h>
 #include <math.h>
 
+#if defined(__amd64__) || defined(_M_AMD64) || defined(__i386__) || defined(_M_X86)
```

Comment:
```suggestion
#ifdef __SSE4_1__
```

Reply:
Does that work for MSVC as well?

From here it looks like it does not:
https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019

Reply:
no